.pipelines/templates/package-helm-chart.yaml

```diff
@@ -1,6 +1,15 @@
 steps: 
+- script: |
+    echo "##vso[task.setvariable variable=UNIQUE_TAG]$(git rev-parse HEAD | cut -c1-13)"
```

Comment:
I think `git rev-parse --short HEAD` should be sufficient here?

Reply:
Yep this works, just tested locally.